creator-node/test/utils.test.js

```diff
@@ -49,7 +48,7 @@ describe('test src/utils.js', () => {
     const ipfsSingleByteCatSpy = sinon.spy(utils, 'ipfsSingleByteCat')
     const ipfsAddSpy = sinon.spy(ipfsLatest, 'add')
 
-    await utils.rehydrateIpfsDirFromFsIfNecessary(multihash, storagePath, logContext)
+    await utils.rehydrateIpfsDirFromFsIfNecessary(multihash, logContext)
```

Comment:
why this change?

Reply:
this is the signature of `rehydrateIpfsDirFromFsIfNecessary`. just fixing the test!
```
async function rehydrateIpfsDirFromFsIfNecessary (dirHash, logContext) {
```

Reply:
wow, nice find. idk how this was working previously lol

Reply:
it just printed out really wonkily :) no impact on the test